### ISSUE TITLE
Root Folder problem

### ISSUE THREAD

**Issue by axxesss** [Lidarr/Lidarr#1160]
I restored a lidarr backup to a new machine that has a different letter root drive in windows 10. I then manually moved all my files to the new root drive (Z:) from the previous drive letter (R:). Once all existing files were moved from (R:) to (Z:) they all work correctly and any new files that are added are sent to the correct artist folder on the correct drive. However, when lidarr adds a new artist lidarr still tries to set the folder up on (R:) . I then have to manually move it to (Z:) and from then on all is well with the newly added artist. I have looked over the config files and cannot seem to find where lidarr keeps the settings for adding new artists. Any help with this issue is much appreciated and thank you to the developers for their outstanding work.

[AB#227](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/227)

**Comment by Turbogame** [comment]
Under Library, click Import and then delete the Root folder for your old location.

Easy peasy!

**Comment by axxesss** [comment]
Thank You for the response. I do not have that option when I click import. I even tried making a dummy folder with my old drive letter. When I did, it appeared as you show above and I deleted it. Now it is not there but every time Lidarr adds a new artist it is added to the drive letter that doesn't exist. All is fine if I manually change directories and add the files to my new drive letter. After that, any new files will go to the proper drive. All other existing directories download to the proper place. It seems like there is a setting in a config file somewhere that needs to be changed. Any idea which file this would be?
